errai-bom/pom.xml

```diff
@@ -61,12 +61,12 @@
   <properties>
     <apache.stanbol.htmlextractor.version>0.10.0</apache.stanbol.htmlextractor.version>
     <appengine.version>1.7.6</appengine.version>
-    <asm.version>5.0.3</asm.version>
+    <asm.version>7.1</asm.version>
```

Comment:
Isn't there needed to use asm 8.0.1. What I read in the release announcement http://blog.isomorphic.com/gwt-2-9-0-has-been-released-what-does-that-mean-for-smart-gwt/